src/code/z_play.c

```diff
@@ -295,9 +295,11 @@ void Gameplay_Init(GameState* thisx) {
     // The emulator constantly checks whether PC is 0x81000000, so this works even though it's not a valid address.
     if ((gEntranceTable[((void)0, gSaveContext.entranceIndex)].scene == SCENE_SPOT09) &&
         gSaveContext.sceneSetupIndex == 6) {
-        osSyncPrintf("エンディングはじまるよー\n"); // "The ending starts"
+        // "Here goes the ending!"
+        osSyncPrintf("エンディングはじまるよー\n");
         ((void (*)())0x81000000)();
-        osSyncPrintf("出戻り？\n"); // "Return?"
+        osSyncPrintf("出戻り？\n");
+        // "Return?"
```

Comment:
I personally prefer to have the comment at the top of the printf, instead of the bottom
```suggestion
        // "Return?"
        osSyncPrintf("出戻り？\n");
```